src/js/pensions/config/form.js

```diff
@@ -755,6 +750,10 @@ const formConfig = {
                 reasonForSeparation: {
                   'ui:title': 'Why did the marriage end?',
                   'ui:widget': 'radio'
+                },
+                dateOfSeparation: currentOrPastDateUI('Date marriage ended'),
+                locationOfSeparation: {
+                  'ui:title': 'Place marriage ended',
```

Comment:
I'm wondering if this makes sense / should be required if the applicant is widowed. @J-Quag?